Testing/Unit/PowerShell/CreateReport/New-Report.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+BeforeAll {
+    Import-Module (Join-Path -Path $PSScriptRoot -ChildPath '../../../../PowerShell/ScubaGear/Modules/CreateReport')
+    New-Item -Path (Join-Path -Path $PSScriptRoot -ChildPath "./CreateReportStubs") -Name "CreateReportUnitFolder" -ErrorAction SilentlyContinue -ItemType Directory | Out-Null
```

Comment:
Consider using test drive.   
https://pester.dev/docs/usage/testdrive

Reply:
That's a great suggestion, I hadn't heard about that. We can definitely implement that and will be a big help.